scripts/tasks.sh

```diff
@@ -784,7 +784,7 @@ function weave_to_flannel_primary() {
     rm /tmp/kubeadm-token
 
     cat > /tmp/kubeadm-join.conf <<- EOM
-apiVersion: kubeadm.k8s.io/v1beta2
+apiVersion: kubeadm.k8s.io/v1beta3
```

Comment:
i dont think we should change this as its not backwards compatible with older kubeadm versions.

Reply:
What's the minimum supported version for kubeadm? `v1beta3` was introduced in [Kubernetes 1.22](https://kubernetes.io/blog/2021/08/04/kubernetes-1-22-release-announcement/#more-secure-control-plane-with-kubeadm).

Reply:
Regardless this seems unrelated to your change and I'm not sure the kustomize patches will apply if we don't use the same api version consistently

Reply:
Ok. I still think we should consider cleaning up the deprecation warnings as part of separate effort